add-product.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Add Product</title>
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+
+    <div id="add-prod-div">
+        <h2>Add Product</h2><br>
+        <form id="add-prod-form">
+            <div><label for="prod-name">Name:</label></div>
+            <div><input class="inp-elemnt" id="name" type="text" name="prod-name"></div>
+            <div><label for="desc">Description:</label></div>
+            <div><textarea class="textarea" id="desc" name="desc" rows="6" cols="25"></textarea></div>
+            <div><label for="price">Price:</label></div>
+            <div><input class="inp-elemnt" id="price" type="text" name="price"></div>
+            <div><label for="category">Category:</label></div>
+            <div><select class="inp-elemnt" id="category" name="category">
+                <option value="0" disabled selected> - Choose Category</option>
+                <option value="1">Women</option>
+                <option value="2">Men</option>
+                <option value="3">Children</option>
```

Comment:
Could you add more categories like : electronics, beauty, fashion, toys and games,  furniture...etc, 
because there will be some products not specified for only (women or men or kids).

Reply:
OK i will add them soon